manifests/e/EclipseAdoptium/Temurin/8/8.0.302.8/EclipseAdoptium.Temurin.8.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: EclipseAdoptium.Temurin.8
+PackageVersion: 8.0.302.8
+PackageLocale: en-US
+LicenseUrl: https://adoptium.net/about.html
+Moniker: temurin8
+Tags:
+- openjdk
+- java
+- jvm
+- jdk
+- temurin
+PackageUrl: https://adoptium.net
+Publisher: Eclipse Adoptium
```

Comment:
```suggestion
Publisher: Eclipse Foundation
```

Reply:
Vendor name has changed to `Eclipse Adoptium` instead of `Eclipse Foundation` that was one of the premises for this change.